src/main/telemetry/msp_shared.h

```diff
@@ -27,3 +27,6 @@ typedef union mspTxBuffer_u {
 void initSharedMsp(void);
 bool handleMspFrame(uint8_t *frameStart, uint8_t *frameEnd);
 bool sendMspReply(uint8_t payloadSize, mspResponseFnPtr responseFn);
+void initMspRingBuffer(void);
```

Comment:
What is the reason that this is exported and not just called inside `initSharedMsp()`?

Reply:
Don't want to reset the ring buffer when a new frame comes in.

Reply:
Ha, overlooked that one.